.vscode/workspace.code-workspace

```diff
@@ -16,6 +16,18 @@
             "name": "scan-request-sender",
             "path": "../packages/scan-request-sender"
         },
+        {
+            "name": "web-api-scan-request-sender",
```

Comment:
Can we have some other name? web-api is not related I think. Maybe wcp, on-demand, continues, stream, batch or something more related?

Reply:
i ddidnt want to use the word wcp (the consumer), as it could change. instead name it that is being used for requests submitted via web api

Reply:
that can be used not only by web api and it is going to be main workflow after moving to full wcp support. web-api is less appropriate here. Call main- at least